addons/udes_stock/controllers/stock_picking.py

```diff
@@ -76,6 +76,16 @@ def is_compatible_package(self, ident, package_name):
 
         return picking.is_compatible_package(package_name)
 
+
+    @http.route('/api/stock-picking/<ident>/can_handle_partials',
+                type='json', methods=['GET'], auth='user')
+    def can_handle_partials(self, ident):
+        """ Check if the picking can handle partials """
+        Picking = request.env['stock.picking']
+        picking = Picking.browse(int(ident))
+        return picking.can_handle_partials()
```

Comment:
Could you update the udes_stock readme to include documentation of the new route?